packages/ui/src/components/Stepper/index.tsx

```diff
@@ -95,101 +183,170 @@ const StyledContainer = styled.div<{
 `
 
 type StepperNumbersProps = {
-  temporal: Temporal
-  children: ReactNode
-  CurrentStep: number
-  size?: 'small' | 'medium'
+  onClick?: (index: number) => void
+  index: number
+  disabled?: boolean
+  title?: ReactNode
+  children?: ReactNode
 }
 
-const StepperNumbers = ({
-  temporal,
+const Step = ({
+  index,
```

Comment:
I'm thinking that maybe the index could be guessed and provided in the parent, wdyt?